src/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/Asn1/TbsCertificateAsn.manual.cs

```diff
@@ -24,44 +24,8 @@ namespace System.Security.Cryptography.X509Certificates.Asn1
     //     extensions      [3]  Extensions OPTIONAL
     //                          -- If present, version MUST be v3 --
     // }
-    [StructLayout(LayoutKind.Sequential)]
-    internal sealed class TbsCertificateAsn
+	internal partial struct TbsCertificateAsn
```

Comment:
The whitespace on this line seems off

Reply:
Yeah, VS4Mac sometimes keeps insisting on using tabs instead of spaces. I use VS Code most of the time to avoid it, but I missed this one apparently. Thanks for noticing.